c2corg_api/views/user.py

```diff
@@ -116,20 +120,128 @@ def post(self):
             log.warning('Error persisting user', exc_info=True)
             raise HTTPInternalServerError('Error persisting user')
 
-        # also create a version for the profile
-        DocumentRest.create_new_version(user.profile, user.id)
```

Comment:
OK, in this case I will move the whole creation of the user profile and first version to the email validation code and it will be easier to remove delayed users. I will do the change when implementing the cleaning part.